docs/publishing/sections/8_ckan.md

```diff
@@ -15,42 +16,50 @@ other agencies, Caltrans publishes [many data sets](https://data.ca.gov/organiza
 Data is generally published as flat files (typically CSV) alongside required
 metadata and a data dictionary.
 
-## Cal-ITP data sets
-* [Cal-ITP GTFS Schedule Data](https://data.ca.gov/dataset/cal-itp-gtfs-ingest-pipeline-dataset)
+## Cal-ITP datasets
+
+* [Cal-ITP GTFS-Ingest Pipeline Dataset (schedule data)](https://data.ca.gov/dataset/cal-itp-gtfs-ingest-pipeline-dataset)
 
 ## General process
+
 ### Develop data models
+
 Generally, data models should be built in dbt/BigQuery if possible. For example,
-we have [latest-only GTFS schedule models](https://github.com/cal-itp/data-infra/tree/main/warehouse/models/gtfs_schedule_latest_only)
+we have [latest-only GTFS schedule models](https://github.com/cal-itp/data-infra/tree/main/warehouse/models/mart/gtfs_schedule_latest)
 we can use to update and expand the existing [CKAN dataset](https://data.ca.gov/dataset/cal-itp-gtfs-ingest-pipeline-dataset)
-[latest-only agency](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.agency)
 
 ### Document data
+
 California Open Data requires two documentation files for published datasets.
+
 1. `metadata.csv` - one row per resource (i.e. file) to be published
 2. `dictionary.csv` - one row per column across all resources
 
-If you are using dbt exposure-based data publishing, you can automatically generate
+We use dbt exposure-based data publishing to automatically generate
 these two files using the main `publish.py` script (specifically the `generate-exposure-documentation`
 subcommand). The documentation from the dbt models' corresponding YAML will be
 converted into appropriate CSVs and written out locally.
 
 Run this command inside the `warehouse` folder, assuming you have local dbt
 artifacts in `target/` from a `dbt run` or `dbt compile`.
+
 ```bash
 poetry run python scripts/publish.py document-exposure california_open_data
 ```
 
+Each day, new versions of `metadata.csv` and `dictionary.csv` are also automatically generated for tables in the production warehouse by the `dbt_run_and_upload_artifacts` job in [the `transform_warehouse` DAG](https://o1d2fa0877cf3fb10p-tp.appspot.com/dags/transform_warehouse/grid), and placed inside the `calitp-dbt-artifacts` GCS bucket.
```

Comment:
I don't think this is actually the case; _dbt_ artifacts are generated but CKAN-specific files are only created via `publish.py`. It'd be appropriate to say these are created weekly by `publish_california_open_data`.

Reply:
You're right; I meant to show here that the `manifest.json` is used to feed into the publication script, not `metadata.csv` and `dictionary.csv`. Will fix in the next commit.

Reply:
This language is still incorrect, as the CSVs are not uploaded by this job

Reply:
Actually resolved in [bbab76f](https://github.com/cal-itp/data-infra/pull/2839/commits/bbab76f6beef1c90d43baae756c8c0de98a50795)